app/helpers/companies_helper.rb

```diff
@@ -0,0 +1,37 @@
+module CompaniesHelper
+  def sort_companies_by_city?
+    if params[:sort]
+      @sorted_by_city_note = "Sorted by city alphabetic order:"
+      @companies = Company.all.sort_by_city
```

Comment:
if `sort_by_city` is a class method, we can just call it on `Company` directly without calling the all! `Company.sort_by_city`